src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_constants.py

```diff
@@ -173,9 +173,11 @@ def _current_frames():
 IS_PY310_OR_GREATER = sys.version_info >= (3, 10)
 IS_PY311_OR_GREATER = sys.version_info >= (3, 11)
 IS_PY312_OR_GREATER = sys.version_info >= (3, 12)
+IS_PY313_OR_GREATER = sys.version_info >= (3, 13)
+IS_PY314_OR_GREATER = sys.version_info >= (3, 14)
 
-# Not currently supported in Python 3.12.
-SUPPORT_ATTACH_TO_PID = not IS_PY312_OR_GREATER
+# Not currently supported in Python 3.14.
+SUPPORT_ATTACH_TO_PID = not IS_PY314_OR_GREATER
```

Comment:
Do we know that attach doesn't work in 3.14 for some reason?

Reply:
It might work, but the code in the attach logic explicitly only allows up to 3.13. This is just matching that logic.